src/openfermion/utils/__init__.py

```diff
@@ -35,6 +35,12 @@
 
 from ._trotter_error import error_bound, error_operator
 
+from ._trotter_qasm_conversion import (third_order_trotter_helper,	
```

Comment:
Do we really want to expose helper functions for the public interface? I can't think of a good use for some of them.  We should not import some of the functions we think we will only use as internal helpers in the module, and perhaps mark them with _ to begin.